pkg/server/container_status.go

```diff
@@ -88,3 +93,38 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 		LogPath:     meta.LogPath,
 	}
 }
+
+// toCRIContainerInfo converts internal container object information to CRI container status response info map.
+func toCRIContainerStatusResponseInfo(ctx context.Context, container containerstore.Container, verbose bool) map[string]string {
+	var info map[string]string
+
```

Comment:
```golang
if !verbose {
  return nil
}
```
nit: Less indent is preferred.

Reply:
yeah and I can remember when one return was preferred.. show how old I am :-)